cooltools/cli/pileup.py

```diff
@@ -185,8 +185,14 @@ def pileup(
         view_df = read_viewframe_from_file(view, clr, check_sorting=True)
 
     # make sure feature are compatible with the view_df
-    if not bioframe.is_contained(features_df, view_df):
-        raise ValueError("Features are not contained in view bounds")
+    if features_format.lower() == "bed":
+        if not bioframe.is_contained(features_df, view_df, cols=bed_cols):
+            raise ValueError("Features are not contained in view bounds")
+    else:
```

Comment:
maybe if bedpe instead of else?

Reply:
actually, all of this would disappear if we internally convert to bedpe at the start of this function.